python/python/glide/async_commands/redis_modules/search/commands.py

```diff
@@ -0,0 +1,96 @@
+from typing import List, Optional, cast
+
+from glide.async_commands.redis_modules.search.optional_params import (
+    FieldFlags,
+    Frequencies,
+    Highlights,
+    InitialScan,
+    Offset,
+)
+from glide.constants import TOK
+from glide.protobuf.redis_request_pb2 import RequestType
+from glide.redis_client import TRedisClient
+
+from .field import Field
+from .index import Index
+
+
+class Search:
+    """Class for RediSearch commands."""
+
+    async def create_index(
+        self,
+        client: TRedisClient,
+        index: Index,
+        schema: List[Field],
+        max_text_fields: bool = False,
+        temporary_seconds: Optional[int] = None,
+        offset: Offset = Offset.STORE_OFFSET,
+        highlight: Highlights = Highlights.USE_HIGHLIGHTS,
+        field_flags: FieldFlags = FieldFlags.USE_FIELDS,
+        frequencies: Frequencies = Frequencies.SAVE_FREQUENCIES,
+        stop_words: Optional[List[str]] = None,
+        initial_scan: InitialScan = InitialScan.SCAN_INDEX,
+    ) -> TOK:
+        """
+        Creates a new search index with the specified configuration.
+        See https://redis.io/commands/ft.create/ for more details.
+
+        Args:
+            client (TRedisClient): The Redis client to execute the command.
+            index (Index): The definition of the index, including its name and type.
+            schema (List[Field]): The schema of the index, specifying the fields and their types.
+            max_text_fields (bool): Forces RediSearch to encode indexes as if there were more than 32 text attributes.
```

Comment:
change "RediSearch" to "the module", as it would be available also with VSS or other compatible APIs